### PR TITLE
refactor: encapsulates my offers into a new instance of offers context

### PR DESCRIPTION
**Pull request by farinalvaro**

### CODE REVIEW

**Review by julianmrodri — DISMISSED** [review]
LGTM

File: src/context/Services/storage/offersActions.ts [hunk 1]
```diff
@@ -67,6 +68,11 @@ export const storageOffersActions: Actions = {
       ...payload,
     },
   }),
+  CLEAN_UP_FILTERS: (state, _) => ({
+    ...state,
+    filters: initialState.filters,
+    needsRefresh: true,
+  }),
```
Comment (jurajpiar) [inline src/context/Services/storage/offersActions.ts:75]: Oh. forgot to remove these.

File: src/context/Services/storage/offersActions.ts [hunk 2]
```diff
@@ -39,6 +39,7 @@ export type Actions = {
   FILTER: StorageOffersReducer<FiltersLimits>
   UPDATE_LIMITS: StorageOffersReducer<LimitsPayload>
   CLEAN_UP: StorageOffersReducer<{}>
+  CLEAN_UP_FILTERS: StorageOffersReducer<{}>
```
Comment (jurajpiar) [inline src/context/Services/storage/offersActions.ts:42]: and this

File: src/context/Services/storage/offersActions.ts [hunk 3]
```diff
@@ -4,7 +4,7 @@ import { StorageOrder } from './interfaces'
 import { initialState, OffersListing, StorageOffersState } from './OffersContext'
 
 export type OFFERS_ACTION = 'FILTER' | 'SET_LISTING' | 'REFRESH' | 'UPDATE_LIMITS' | 'SET_ORDER'
-  | 'CLEAN_UP'
+  | 'CLEAN_UP_FILTERS' | 'CLEAN_UP'
```
Comment (jurajpiar) [inline src/context/Services/storage/offersActions.ts:7]: and this)

**Review by jurajpiar — CHANGES_REQUESTED** [review]
just remove the unused code I left there please

**Review by jurajpiar — APPROVED** [review]

**Pull request merged**